### DISCUSSION TITLE
Server update

### DISCUSSION THREAD

**Discussion by leorke** [BearWare/TeamTalk5#2119] [Q&A]
Hello everybody,
is there a possibility of updating a server from a remote station?

**Comment by bear101** [comment]
You can press F9 in TeamTalk for Desktop PCs to update server properties.

**Reply by leorke** [reply]
Thanks! I have another question. I've been having constant disconnections/reconnections recently. Disconnections last a few seconds and reconnection occurs automatically (without any action on my part). Can you help me identify the causes? The server works on a mesh network (AREDN) and is installed in a remote location. Version 5.6.3.5013 runs on the server and 5.14.0.5128 on the client. Could it be due to connection problems with the ethernet cable connecting the server to the router?  I thank you in advance for any help. Regards.

**Reply by bear101** [reply]
What is the server's "User Timeout" setting in "Server Properties" dialog (F9)?

**Comment by leorke** [comment]
User timeout: 60.Leonardo GardinVia L. Da Ponte,2131015 Conegliano - TV - ItalyOff. +39 0438 22529Mob. +39 345 4204536Il giorno 27 dic 2023, alle ore 20:29, Bjørn Damstedt Rasmussen ***@***.***> ha scritto:﻿
What is the server's "User Timeout" setting in "Server Properties" dialog (F9)?

**Comment by leorke** [comment]
Hello Bjorn,

further to your mail, I tried to ping the server and got the following:

“

C:\Windows\System32>ping 10.113.61.101

 

Esecuzione di Ping 10.113.61.101 con 32 byte di dati:

Richiesta scaduta. (request expired)

Richiesta scaduta. (request expired)

Richiesta scaduta. (request expired)

Richiesta scaduta. (request expired)

 

Statistiche Ping per 10.113.61.101:

    Pacchetti: Trasmessi = 4, Ricevuti = 0,

    Persi = 4 (100% persi),

 

C:\Windows\System32>

“

In red a possible translation from Italian 

May be this can help in tracing the issue.

 

Thanks for helping me.

Season’s greetings.

 

Leonardo

 

 

 

Da: Bjørn Damstedt Rasmussen ***@***.***> 
Inviato: mercoledì 27 dicembre 2023 20:30
A: BearWare/TeamTalk5 ***@***.***>
Cc: leorke ***@***.***>; Author ***@***.***>
Oggetto: Re: [BearWare/TeamTalk5] Server update (Discussion #2119)

 

What is the server's "User Timeout" setting in "Server Properties" dialog (F9)?
